github/config.go

```diff
@@ -28,9 +28,13 @@ type Organization struct {
 func (c *Config) Client() (interface{}, error) {
 	var org Organization
 	org.name = c.Organization
-	ts := oauth2.StaticTokenSource(
-		&oauth2.Token{AccessToken: c.Token},
-	)
+
+	var ts oauth2.TokenSource
+	if c.Token != "" {
```

Comment:
Im wondering if now that we have `null` in terraform 0.12 if it might be better to require it to be `null` instead of a string, unfortunately this would break backwards compatibility with < 0.12 though 🤔

Reply:
I thought more about it and I think its important that its a manual opt in for anonymous mode.